### ISSUE TITLE
Save loss and IoU numbers as text file

### ISSUE THREAD

**Issue by maning** [mapbox/robosat#92]
Currently `rs train` saves only history plot and checkpoint pth.  For very large epoch numbers, its hard to understand from the saved plot the small variations in the model after every epoch.  Saving the result as a text file would help dig into the numbers better.

**Comment by daniel-j-h** [comment]
I agree. We could think about integrating more full blown solutions like tensorboard or visdom. But that may be too much for the user-base we are targeting here and the use-cases we have in mind.

Maybe writing out a simple text file you can easily evaluate would be a good idea.

Here are the related code locations where we could add it:

https://github.com/mapbox/robosat/blob/6c7f547d2dd303f8a68b4fbd2ba60583348d7816/robosat/tools/train.py#L77-L98
